PackageVersions.g.props

```diff
@@ -553,5 +553,617 @@ NOTE:  This code was generated by the GeneratePackageVersions tool. To safely
       <Properties>ApiVersion=2.2.0;RestoreRecursive=false;BuildProjectReferences=false</Properties>
       <TargetFramework>netcoreapp2.1</TargetFramework>
     </PackageVersionSample>
+    <PackageVersionSample Include="samples*\**\Samples.Amazon.SQS.csproj">
```

Comment:
Yeah this is a massive list of versions. Maybe we should prioritize your major.minor testing change before merging this 😅